tests/models/bridgetower/test_modeling_bridgetower.py

```diff
@@ -0,0 +1,362 @@
+
+    @slow
+    def test_model_from_pretrained(self):
+        for model_name in BRIDGETOWER_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
+            model = BridgeTowerModel.from_pretrained(model_name)
+            self.assertIsNotNone(model)
+
+    def test_save_load(self):
+        config, input_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+            with torch.no_grad():
+                outputs = model(**input_dict)
+
+            out_2 = self.extract_output(outputs, model_class.__name__)
+            out_2 = out_2.cpu().numpy()
+            out_2[np.isnan(out_2)] = 0
+
+            with tempfile.TemporaryDirectory() as tmpdirname:
+                model.save_pretrained(tmpdirname)
+                model = model_class.from_pretrained(tmpdirname)
+                model.to(torch_device)
+                with torch.no_grad():
+                    after_outputs = model(**input_dict)
+
+                # Make sure we don't have nans
+                out_1 = self.extract_output(after_outputs, model_class.__name__)
+                out_1 = out_1.cpu().numpy()
+                out_1[np.isnan(out_1)] = 0
+                max_diff = np.amax(np.abs(out_1 - out_2))
+                self.assertLessEqual(max_diff, 1e-5)
+
+    def test_determinism(self):
+        config, input_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+            with torch.no_grad():
+                first = model(**input_dict)
+                second = model(**input_dict)
+
+            out_1 = self.extract_output(first, model_class.__name__).cpu().numpy()
+            out_2 = self.extract_output(second, model_class.__name__).cpu().numpy()
+            out_1 = out_1[~np.isnan(out_1)]
+            out_2 = out_2[~np.isnan(out_2)]
+            max_diff = np.amax(np.abs(out_1 - out_2))
+            self.assertLessEqual(max_diff, 1e-5)
+
+    @unittest.skip(
+        reason="""Bridge Tower model currently does not output hidden states. So this test is not applicable."""
+    )
+    def test_hidden_states_output(self):
+        pass
+
+    @unittest.skip(
+        reason="""Bridge Tower model currently does not output hidden states. So this test is not applicable."""
+    )
+    def test_retain_grad_hidden_states_attentions(self):
+        pass
+
+    @unittest.skip(reason="""Bridge Tower does not have input/output embeddings. So this test is not applicable.""")
+    def test_model_common_attributes(self):
+        pass
+
+    @unittest.skip(
+        reason="""Bridge Tower model currently only supports pretrained model. Thus this test is not needed."""
+    )
+    def test_initialization(self):
+        pass
```

Comment:
Please don't skip this test, should pass

Reply:
This test is no longer skipped. Thanks for the suggestion!